tt-metal-yolov4/requirements.txt

```diff
@@ -0,0 +1,6 @@
+# inference server requirements
+fastapi==0.85.1
+uvicorn==0.19.0
+python-multipart==0.0.5
+
+-f https://download.pytorch.org/whl/cpu/torch_stable.html
```

Comment:
What is `torch` being pulled for here?

Reply:
Torch is currently required because the model implementation takes `torch.Tensor`s as inputs and returns them as outputs. https://github.com/tenstorrent/tt-metal/blob/65d246482b3fd821d383a4aa2814f1de5392f417/models/demos/yolov4/tests/yolov4_perfomant_webdemo.py#L343

We need torch on the server-side to perform torch tensor conversions from and to numpy arrays